db/schema.rb

```diff
@@ -54,12 +54,12 @@
     t.text     "comments"
     t.boolean  "adult"
     t.boolean  "appropriate_adult",         default: false, null: false
+    t.string   "state"
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.string   "state"
     t.string   "dscc_number"
-    t.text     "feedback"
     t.datetime "interview_start_time"
+    t.text     "feedback"
```

Comment:
@pixeltrix Since I just pulled the repo and migrated, am I correct in assuming that it is out of sync further upstream?